src/otx/algo/segmentation/huggingface_model.py

```diff
@@ -52,7 +52,7 @@ class HuggingFaceModelForSegmentation(OTXSegmentationModel):
     Example:
         1. API
             >>> model = HuggingFaceModelForSegmentation(
-            ...     model_name_or_path="nvidia/segformer-b0-finetuned-ade-512-512",
+            ...     model_version="nvidia/segformer-b0-finetuned-ade-512-512",
```

Comment:
Actually, I think this is more of a `name` than a `version`, but what do you think?

Reply:
Yes, but the other models and OTXSegmentationModel has "model_version" naming. I think we need to align this naming. We can change model_version to model_name everywhere though

Reply:
Hmm, but i think it's a little bit differen to have multiple versions of one architecture and to provide multiple architectures in one framework. In the case of the hugging-face example, different versions have different architectures, and those architectures can have multiple versions.

Reply:
I decided to go with "model_name" everywhere since "lite_hrnet_x" or "segnext_small" are more names, than versions. "model_version" seems to be confusing as parameter name in OTXSegmentationModel, but model_name looks good. Also, HuggingFace and DINOV2 look better with "model_name". 
So, I propose to keep this naming :)